postgresql/PgDatabase.js

```diff
@@ -156,7 +158,7 @@ class PgDatabase {
 					);
 				}
 
-				return Promise.allSettled(promises);
+				return Promise.all(promises);
```

Comment:
code didn't check if all went ok before.

So I was debugging why my new table wasn't created since I saw no errors in console.

It also turned out that previously used docker image for postgres didn't have postgis extension:
```
#ERROR# error: type "geometry" does not exist
```
but before this change it was silently ignored. After this change, ci failed (https://github.com/gisat/puma/runs/720512777?check_suite_focus=true).